gce/provision.sh

```diff
@@ -1,32 +1,65 @@
 #!/bin/bash
```

Comment:
One problem with this script is that it does not stop if there is an error. This is not an issue for this PR, ofc, but we should fix that.